src/rules/no-array-length-in-deps.ts

```
@@ -0,0 +1,276 @@
+					spec.imported.name === 'stableHash'
+				) {
+					return true;
+				}
+			}
+		}
+	}
+	return false;
+}
+
+export const noArrayLengthInDeps = createRule<[], MessageIds>({
+	name: 'no-array-length-in-deps',
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)